src/cljs/hello_world/game.cljs

```diff
@@ -176,7 +176,9 @@
               bottom-button
               (fn []
                 (when (util/currently-playing-game?)
-                  (sound/play-beep! 300)
+                  (sound/play-beep! (sound/frequencies-in-major-scale-4th-octave
+                                      (mod (+ 1 util/row-col-num col)
+                                           (count sound/frequencies-in-major-scale-4th-octave))))
```

Comment:
1+5+0 =  6 / 8 ?
1 +5 + 1 = 7 / 8 ?? How dose it work? :)

Reply:
```clojure
(mod (+ 1 5 0) 8)
=> 6
(mod (+ 1 5 1) 8)
=> 7
(mod (+ 1 5 2) 8)
=> 0
(mod (+ 1 5 3) 8)
=> 1
```
^ Is this enough example to understand how mod works?

Reply:
When first argument is negative
```clojure
(mod -6 8)
=> 2
(mod -7 8)
=> 1
(mod -8 8)
=> 0
```